vignettes/Introduction_agestructEPICYST.Rmd

```diff
@@ -0,0 +1,164 @@
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, echo=FALSE}
+library(epicyst)
+```
+
+# Call default parameters and equilibrium solutions
+
+The `set_up()` function provides the user a means to specify parameters and starting conditions for a model run. Input parameters, as well as starting prevalence for Taeniasis, and Cysticercosis (in the porcine and human populations) may be left at the default values or specified in the function call. 
+
+Changes to default prevalence values will initiate to re-estimation of transmission parameters to allow the system to run at the newly specified endemic equilibrium. The `set_up()` function returns a list of two elements: 1) The input parameters for a model run (standardised time units and dependent parameters included) and 2) Initial state variable values.
+
+To initiate and view default parameters:
+
+```{r}
+initialise <- set_up()
+```
+
+```{r}
+# View parameters
+initialise[[1]][1:6]
+# View state variable starting values (susceptible humans and pigs)
+initialise[[2]][2]
+initialise[[2]][5]
+```
+
+Input arguments for `set_up()` are documented in the help file `?set_up()`. Certain combinations of input prevalence may not be possible to fit, in which case an error message will be thrown.
+
+## Age-structured model inputs for human and pig demographics
+
+In the age-structured model, human and pig compartments are stratified into age groups. For pigs, there are 150 age groups (one month per group) in the default model, which can be viewed (and modified) through the following `set_up(number_age_classes_pig = 150)` for the number of pig age classes, and `set_up(pig_age_class_width = 1)` for the width (in months) of these age group.
+
+For humans in the default model, there are 7 age groups, representing 0 - 4.99, 5 - 9.99, 10 - 14.99, 15 - 29.99, 30 - 49.99, 50 - 69.99 and 70 - 89.99 years. These age groups are structured to include school-age children (5 - 9.99 & 10 - 14.99 yrs). While there is an input to change these `set_up(number_age_classes_human = 7)`, leaving this default model for human age-structuring is recommended. 
+
+Figure 1 below provides a representation of the age-structure in the model, including demographic and transmission parameters. 
+
+```{r echo=FALSE, out.width='100%'}
+knitr::include_graphics('./EPICYST_schematic_legend.jpg')
+```
+
+## Other key input parameters in the age-structured pig model
+
+Input parameters into `set_up()` specify additional age-specific features in the pig age-structured model. Firstly, the `set_up(slaughter_age_min = 6)` input
+specifies the minimum age (in months) in which pigs are slaughtered, which is set to 6 months in the default model. Secondly, the `set_up(slgEP = 1)` specifies the number of years pigs are expected live, on average, after reaching slaughter age (a default for the average natural life expectancy for pigs is also specified by `set_up(LEP = 10)`). 
```

Comment:
could we change all the example calls (e.g. `set_up(slgEP = 1)`) to just have the input, otherwise it might prompt the user to call `set_up()` multiple times. So you could just put `slgEP = 1` in the example instead